src/main/kotlin/coverosR3z/logging/Logger.kt

```diff
@@ -5,5 +5,7 @@ import coverosR3z.authentication.CurrentUserAccessor
 val cua = CurrentUserAccessor()
 
 fun logInfo(msg : String) {
-    println("INFO: ${cua.get().name}: $msg")
+    try {
```

Comment:
I think there might be a more elegant way to handle this.

Reply:
Yeah, this was purely for getting through to the tests. Need to think about cua.get() failing more gracefully

Reply:
There might be a way to segregate our logging - between that which is logging commands by a user versus being controlled by the system.  Or maybe when someone logs in the user is "system"?  Let's discuss.

Reply:
Easiest option: User-aware logging wrapping regular logging and prepending `$user:`
Second easiest: nullable logging/cua.get() call that quickly decides whether it has a user or whether it is `SYSTEM` or something of the like